### ISSUE TITLE
pressSound/hoverSound functions not working...(bug?)

### ISSUE THREAD

**Issue by SniperHauss89** [lumien231/Custom-Main-Menu#205]
Hello, I have been successfully creating a custom main menu for a couple of weeks but as of a couple days ago I ran into an issue with the pressSound and hoverSound functions. 
   I have it setup exactly like GreatOrator in his tutorial from a while ago https://www.youtube.com/watch?v=-qUEBXbalH8 with no success. I did determine via some testing that it's not something wrong with my setup and that the pack he uses button sounds in is for 1.7.10 and I am using 1.10.2 so I set up a 1.7.10, 1.8.9, 1.9.4, 1.10.2, and 1.12 forge instance on my vanilla mc launcher and tested each version and 1.7.10 and 1.8.9 work fine but the feature stops working with 1.9.4 (both versions) and still doesn't work with 1.10.2 on up to 1.12..... is this a bug that's gone undetected due to the feature not being used?
   In conclusion I have concluded that it is related to either the mod itself or resource loader and possibility a slim chance of being forge for versions 1.9.4 and up. I am perplexed as to why it isn't working but would love to still utilized the feature so any further help with be great.

My sounds JSON:
   {
	"click": {"sounds": ["click"]},
	"hover": {"sounds": ["hover"]}
   }

Custom Main Menu JSON:
               "action" :
            {
                "type" : "openGui",
                "gui" : "languages"
            },
	       "hoverSound" : "menu:hover",
	       "pressSound" : "menu:click"

and the path to the sounds folder and JSON file: \resources\menu (sound folder and JSON in the menu folder).

**Comment by Skyler0x2** [comment]
I have the same issue

**Comment by lumien231** [comment]
Got it to work, not sure whether it was different in older versions but in your sounds.json your lines should look like this:

> "click":{"sounds":["menu:click"]}

Otherwise mc will look for the sound in its own resources.

**Comment by SniperHauss89** [comment]
Oh nice been busy and didn't see your reply but thanks a bunch @lumien231 for figuring that out it was driving me nuts lol.
